scripts/refresh-sharding-and-restart

```diff
@@ -2,8 +2,16 @@
 
 set -e
 
-"$(dirname "$0")/zulip-puppet-apply" -f
-# The step above should have generated the config files, now we need to move them into place:
+# Stand up the new zulip-tornado supervisor instances, and write out
+# the new generated the config files, with .tmp
```

Comment:
```suggestion
# the newly generated config files, with .tmp suffix
```

I think something like that was the intended phrasing?

Reply:
Absolutely!  Thanks for the fixup.